portable/NetworkInterface/STM32Fxx/NetworkInterface.c

```diff
@@ -63,7 +65,6 @@
     #error What part?
 #endif /* if defined( STM32F7xx ) */
 
-#include "stm32fxx_hal_eth.h"
```

Comment:
I have not yet seen compiler warnings or errors here, but I believe that you did.

Reply:
Yes, I created a new project and there were warnings and errors.